apps/web-mzima-client/src/app/shared/components/support-modal/support-modal.component.ts

```diff
@@ -38,7 +38,7 @@ export class SupportModalComponent extends BaseComponent {
         title: this.translate.instant('app.documentation.title'),
         description: this.translate.instant('app.documentation.description'),
         action: () => {
-          this.openUrl('https://docs.ushahidi.com/ushahidi-platform-user-manual');
+          this.openUrl('https://docs.ushahidi.com/platform-user-manual/v/mzima');
           this.closeModal();
```

Comment:
Please remove this file so that only the readme file change shows up in this PR. You have 2 files changed instead of 1 @MMwandigha

Reply:
@Ifycode Ok